pkg/dockerregistry/server/registry.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import (
+	"github.com/docker/distribution"
+	"github.com/docker/distribution/context"
+	"github.com/docker/distribution/registry/middleware/registry"
+)
+
+// Registry represents a collection of repositories, addressable by name.
+// This variable holds the object created by the docker/distribution. We import
+// it into our namespace because there are no other ways to access it. In other
+// cases it is hidden from us.
+var Registry distribution.Namespace
```

Comment:
Not true, go is package-based. Which means you can access everything from an entire package. So files do not matter as long as they're in the same directory. Check our current `pkg/dockerregistry` packge for examples.